package-lock.json

```diff
@@ -15069,9 +15081,15 @@
       "integrity": "sha512-YATxVxgRqNH6nHEIsvg6k2Boc1JHI9ZbH5iWFFv/MTkchz3b1ieGDa5T0a9RznNdI0KhVbdbWSN+KWWrQZRxTw=="
     },
     "@types/node": {
+<<<<<<< HEAD
```

Comment:
I think some of these additions, like this one and others below, are artifacts from a rebase and need to be removed